manifests/repo.pp

```diff
@@ -3,7 +3,7 @@
 #
 class jenkins::repo(
   Stdlib::Httpurl $base_url = 'https://pkg.jenkins.io',
-  String $gpg_key_filename = 'jenkins.io.key',
+  String $gpg_key_filename = 'jenkins.io-2023.key',
```

Comment:
is the old jenkins instance affected by this modification?

Reply:
Only affects when you install a new Jenkins package.